lib/Simulator/Simulator.cpp

```diff
@@ -83,18 +112,15 @@ void SimulatorClass::Main()
       //Pending record doesn't contain valid data, this means we've reched the end of our queue
       return;
    }
+   long lRaceElapsedTime = GET_MICROS - RaceHandler._lRaceStartTime;
    //Simulate sensors
-   if (RaceHandler.RaceState != RaceHandler.STOPPED
-      && PendingRecord.lTriggerTime <= (long) RaceHandler._lRaceTime)
+   if (RaceHandler.RaceState != RaceHandler.STOPPED && PendingRecord.lTriggerTime <= (long)lRaceElapsedTime)
```

Comment:
Why did you change this? `RaceHandler._lRaceTime` is continuously calculated in `RaceHandler.Main()`, so you're doing the same calculation here again (on line 115), but if you use `RaceHandler._lRaceTime` you get the same number.

Reply:
Without this change (that was in original release for Arduino) simulator doesn't work for cases/races with "false start" as it subtract negative triggers (e.g. -102316) from "race start time" that is not yet updated and is equal to time of pressing START button still. Effect is that fault instead be seen 102ms before green light, it's indicated 102ms after START sequence is triggered. I hope you got the point.

Reply:
Clear, thanks!